### DISCUSSION TITLE
How do I update filebeat ingest pipelines on search node?

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#5902] [Unsupported Versions] [answered]
SO 2.3.70 - Distributed - ISO

Hello, I'm trying to fix a log.level string/long type index conflict that I think is caused by the filebeat-7.13.4-azure-signinlogs-pipeline and filebeat-7.13.4-azure-auditlogs-pipeline pipelines.

"azure.signinlogs.Level" & "azure.auditlogs.Level" fields contain numbers and those fields are renamed to log.level, in the respective pipelines.  This is where I'm focusing my blame, but I'm still learning how all this magic works...

Using Kibana on my manager server, I replaced the renaming of the field with converting to string/renaming as follows:

Old value:

    "rename" : {
      "field" : "azure.auditlogs.Level",
      "target_field" : "log.level",
      "ignore_missing" : true
    }

New value:

    "convert": {
      "field": "azure.auditlogs.level",
      "type": "string",
      "target_field": "log.level"
    }

When I run the following on my search node, I'm still seeing the old pipeline

    curl -k --user (creds redacted) -X GET "https://localhost:9200/_ingest/pipeline/filebeat-7.13.4-azure-auditlogs-pipeline?pretty"

The search node shows connected under remote clusters (I had to add the IP/name of the search node to /etc/hosts on manager to make get it connected)

I've restarted elasticsearch on the search node a couple of times using so-elasticsearch-restart thinking that might help, but nope.  

A kick in the right direction will be most appreciated.

Thank you, ...Rob

**Comment by palichx** [comment, marked as answer]
Hi,
ingest pipeline directory is (/opt/so/saltstack/default/salt/elasticsearch/files/ingest/) /opt/so/saltstack/local/salt/elasticsearch/files/ingest/.
Put correct pipeline file hear, and then restart elastic on search node.

**Reply by Rbartram** [reply]
Thank you.
